### ISSUE TITLE
Building NVDA for 64 bit fails on Windows 7 32 bit, MSVC 2010

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2508]
**Reported by mherrmann.at on 2012-07-05 16:11**
I'm having trouble building the 64 bit version of the current development version of NVDA. My system is:
 * Windows 7 Professional SP 1 32 bit
 * Visual Studio 2010 Professional. 

Building for 32 bit works fine:
```
scons dist targetArchitectures=x86
```
When I just do
```
scons dist
```
then I get an error dialog saying "The program can't start because mspdb100.dll is missing from your computer. Try reinstalling the program to fix this problem.". Calling vcvarsall using the command
```call "c:\Program Files\Microsoft Visual Studio 10.0\VC\vcvarsall.bat"```
gets rid of this error dialog, however results in a lot of unresolved externals (see attached file "unresolved externals.txt").

All problems go away when I comment out the following the following lines in site_scons\site_tools\windowsSdk.py:
```
    ...
    #msvc.msvc_setup_env_once=lambda env: False
    #msvc.msvc_exists=lambda: True
    ...
    #env['MSVC_USE_SCRIPT']=False
    ...
```
This change is in the attached file "windowsSdk_fixed.patch".

Should I switch to a different version of Visual Studio? I failed to find any documentation on the recommended Visual Studio version.

**Comment by jcsteh** [comment]
No longer relevant, as we now only support Visual Studio 2015 and there's no longer a separate SDK tool chain.
